packages/pcd-types/src/pcd.ts

```diff
@@ -209,6 +224,7 @@ export enum ArgumentTypeName {
   StringArray = "StringArray",
   PCD = "PCD",
   ToggleList = "ToggleList",
+  Identity = "Identity",
```

Comment:
All the types here should be trivially `JSON.serialize`-able. Otherwise they would not be able to be robustly serialized and then deserialized, which is what happens to the arguments data structure at the interface between third party applications and zupass.